cpp/nacjd.cc

```diff
@@ -0,0 +1,360 @@
+/** \file    nacjd.cc
+    \brief   Identifies URL's that we can use for further processing..
+    \author  andreas-ub
+
+    \copyright 2021 Universitätsbibliothek Tübingen
+
+    This program is free software: you can redistribute it and/or modify
+    it under the terms of the GNU Affero General Public License as
+    published by the Free Software Foundation, either version 3 of the
+    License, or (at your option) any later version.
+
+    This program is distributed in the hope that it will be useful,
+    but WITHOUT ANY WARRANTY; without even the implied warranty of
+    MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+    GNU Affero General Public License for more details.
+
+    You should have received a copy of the GNU Affero General Public License
+    along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+#include <algorithm>
```

Comment:
The order is still wrong. getopt.h, or for that matter, anything ending in .h is not a C++ header.